src/index.html

```diff
@@ -11,7 +11,154 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field vertical">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field vertical">
+          How old are You?
+          <input
+            type="number"
+            name="howold"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+
+        <label class="form-field vertical">
+          Full date of birth:
+          <input
+            type="date"
+            name="date"
+          >
+        </label>
+
+        <label class="form-field vertical">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="form-field vertical">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="8"
+            maxlength="32"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="yes"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input type="radio" name="no">
+            No
+          </label>
```

Comment:
fix this
<img width="229" alt="image" src="https://user-images.githubusercontent.com/53611910/216794465-17a1b45b-9775-42b8-8a3b-e7add6d85ad1.png">